dbms/src/TestUtils/MPPTaskTestUtils.h

```diff
@@ -20,7 +20,7 @@
 
 namespace DB::tests
 {
-DAGProperties getDAGPropertiesForTest(int server_num);
+DAGProperties getDAGPropertiesForTest(int server_num, int local_query_id, int tidb_server_id);
```

Comment:
```suggestion
DAGProperties getDAGPropertiesForTest(int server_num, int local_query_id = -1, int tidb_server_id = -1);
```
Then there is no need to heavily modify the code that uses `getDAGPropertiesForTest`

Reply:
OK, updated